hydra-main/src/main/java/com/addthis/hydra/minion/JobTask.java

```diff
@@ -789,7 +794,7 @@ public void exec(@Nonnull CommandTaskKick kickMessage, boolean execute) throws E
             String jobId = kickMessage.getJobUuid();
             int jobNode = kickMessage.getJobKey().getNodeNumber();
             log.debug("[task.exec] {}", kickMessage.getJobKey());
-            require(testTaskIdle(), "task is not idle");
+            require(testTaskIdle(), "task is not idle", ExecStateException::new);
```

Comment:
It'd be better to include the actual non-idle state, perhaps job id and task id too, in the exception message